AndroidPush/Apps/push-tutorial-two/build.gradle

```diff
@@ -0,0 +1,47 @@
+apply plugin: 'com.android.application'
```

Comment:
So why doesn't the second tutorial start on a separate branch? It shouldn't depend on any of the earlier commits relating to the device-based registration